lmcache/experimental/storage_backend/connector/mooncakestore_connector.py

```diff
@@ -115,11 +116,22 @@ async def exists(self, key: CacheEngineKey) -> bool:
     async def get(self, key: CacheEngineKey) -> Optional[MemoryObj]:
         key_str = key.to_string()
 
-        metadata_bytes = self.store.get(key_str + "metadata")
+        try:
+            buffer = self.store.get(key_str) # buffer is pybind11 buffer type
+        except Exception as e:
```

Comment:
I guess you want to log the exception? And what is the expected behavior if `key_str` absent? Return None?